### PR TITLE
remove non applicable .gitignore

### PR DESCRIPTION
**Pull request by mikebrow**
Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,5 +11,5 @@ release-tool
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+# Project-local glide cache, RE: https://github.com/org_name/glide/issues/736
```
Comment (AkihiroSuda) [inline .gitignore:14]: 404

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
404

File: .gitignore [hunk 2]
```diff
@@ -11,5 +11,5 @@ release-tool
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+# Project-local glide cache, RE: https://github.com/github/gitignore/pull/2253
 .glide/
```
Comment (AkihiroSuda) [inline .gitignore:15]: Why do we need this? We don’t use glide.

Comment (mikebrow) [reply .gitignore:15]: no reason that I can discern... https://github.com/github/gitignore/pull/2505/files shows it coming out ... presume we picked it up by default.. will remove.

**Review by AkihiroSuda — APPROVED** [review]

**Review by fuweid — APPROVED** [review]
LGTM

**Pull request merged**